commit_txn.c

```diff
@@ -0,0 +1,90 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <errno.h>
+
+/*
+ * The WT_SESSION structure is a public-facing structure that provides an
+ * abstraction of a thread context for users of WiredTiger.
+ */
+struct __wt_session {
+  char *name;
+  int content;
+};
+typedef struct __wt_session WT_SESSION;
+
+/*
+ * The WT_SESSION_IMPL structure contains a public WT_SESSION structure, and
+ * adds additional fields that are only used internally by WiredTiger.
+ */
+struct __wt_session_impl {
+  WT_SESSION impl;
```

Comment:
I was wondering if the order in which these variables are defined matter, especially `WT_SESSION impl;`.